src/main/java/com/internet/shop/model/Role.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.shop.model;
+
+public class Role {
+    private Long id;
+    private RoleName roleName;
+
+    public Role(RoleName roleName) {
+        this.roleName = roleName;
+    }
```

Comment:
Maybe it is better to define this constructor as private and use of()?

Reply:
Done